doc/letterplace.doc

```diff
@@ -205,10 +209,10 @@ ideal I = x*y + y*z, x*x + x*y - y*x - y*y;
 option(redSB); option(redTail);
 ideal J = std(I);
 J;
-// ----------------------------------
-lp2lstr(J,r); // export an object called @code{@LN} to the ring r
-setring r;  // change to the ring r
-lst2str(@LN,1); // output the string presentation
+@c // ----------------------------------
```

Comment:
@c cannot be used inside examples(would be passed to Singular): delete these lines

Reply:
ok, will be removed